pkg/ko/publish/options.go

```diff
@@ -72,3 +72,11 @@ func WithNamer(n Namer) Option {
 		return nil
 	}
 }
+
+// WithTags is a functional option for overriding the image tags
+func WithTags(tags []string) Option {
```

Comment:
I think we want to use this in `makePublisher` as well. We just need to pass the TagsOptions through to it.

Reply:
added this on the `makePublisher` function.